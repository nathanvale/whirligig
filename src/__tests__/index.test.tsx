import React from 'react'
import {render} from '@testing-library/react'
import {Button} from '../button'

test('renders a styled button with text', () => {
  const testMessage = 'Test Message'
  const {container} = render(<Button>{testMessage}</Button>)
  expect(container.firstChild).toMatchSnapshot()
})
