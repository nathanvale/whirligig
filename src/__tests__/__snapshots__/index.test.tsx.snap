// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a styled button with text 1`] = `
<button
  class="button__Button-w004gt-0 fLSaDs"
>
  Test Message
</button>
`;
